docker/Dockerfile.tcf-dev

```diff
@@ -145,35 +145,41 @@ ARG TCF_DEBUG_BUILD
 ARG DISPLAY
 ARG XAUTHORITY
 
-#TODO need to be removed latter
+#TODO need to be removed later
```

Comment:
I guess this TODO line can be removed. Not sure why is this still here? @pankajgoyal2 @Ram-srini ?